charts/authentik/templates/worker-deployment.yaml

```
@@ -0,0 +1,155 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-%s" (include "authentik.names.fullname" $) "worker" }}
+  labels:
+    {{- include "authentik.labels" $ | nindent 4 }}
+    app.kubernetes.io/component: "worker"
+  {{- if $.Values.annotations }}
+  annotations:
+    {{- toYaml $.Values.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ $.Values.worker.replicas }}
+  selector:
+    matchLabels:
+      {{- include "authentik.labels.selectorLabels" $ | nindent 6 }}
+      app.kubernetes.io/component: "worker"
+  template:
+    metadata:
+      labels:
+        {{- include "authentik.labels.selectorLabels" $ | nindent 8 }}
+        app.kubernetes.io/component: "worker"
+        app.kubernetes.io/version: "{{ $.Values.image.tag }}"
+      annotations:
+        goauthentik.io/config-checksum: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+      {{- if $.Values.podAnnotations }}
+        {{- toYaml $.Values.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec:
+      {{- if $.Values.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml $.Values.image.pullSecrets | nindent 8 }}
+      {{- end }}
+      {{- if $.Values.serviceAccount.create }}
+      serviceAccountName: {{ include "authentik.names.fullname" $ }}
+      {{- end }}
+      {{- with $.Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with $.Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with $.Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      enableServiceLinks: true
+      {{- if $.Values.initContainers }}
```

```suggestion
      {{- if $.Values.initContainers }}
```
Same here, do we want to scope that under `.Values.worker`

---

And same for `additionalContainers`